ui/cryptomaterial/util.go

```diff
@@ -136,3 +138,21 @@ func AnyKeyWithOptionalModifier(modifier key.Modifiers, keys ...string) key.Set
 	keysWithModifier := fmt.Sprintf("(%s)-[%s]", modifier, keysCombined)
 	return key.Set(keysWithModifier)
 }
+
+func UniformPadding(gtx layout.Context, body layout.Widget) layout.Dimensions {
+	width := gtx.Constraints.Max.X
+
+	padding := values.MarginPadding24
+
+	if (width - 2*gtx.Dp(padding)) > gtx.Dp(values.AppWidth) {
+		paddingValue := float32(width-gtx.Dp(values.AppWidth)) / 4
+		padding = unit.Dp(paddingValue)
+	}
+
+	return layout.Inset{
+		Top:    values.MarginPadding24,
+		Right:  padding,
+		Bottom: values.MarginPadding24,
+		Left:   padding,
+	}.Layout(gtx, body)
+}
```

Comment:
We already have this in components.go. I think the thing would be to move it to Theme.go so it can be accessed globally.